MapleServer2/PacketHandlers/Common/ResponseVersionHandler.cs

```diff
@@ -14,6 +15,9 @@ public class ResponseVersionHandler : CommonPacketHandler {
         public ResponseVersionHandler(ILogger<ResponseVersionHandler> logger) : base(logger) { }
 
         public override void Handle(LoginSession session, PacketReader packet) {
+            // Sync the account state TickCount
+            AccountStorage.TickCount = Environment.TickCount;
```

Comment:
Theres a ServerTick at GameSession.cs that does this, in any case are the same, you can use session.ServerTick , or there's a reason to make it in another class?

Reply:
It could be moved as a new property in `GameSession`, and then when `NewCharacter` is called it would need to have the `CreationTime` passed as a property.

Reply:
After experimentation, unfortunately it cannot be moved to `GameSession`, it would need to be pulled from `LoginSession` which breaks current implementation of generics for handling packets.